android/build.gradle

```diff
@@ -22,7 +22,7 @@ apply plugin: 'com.android.application'
 
 android {
 	ndkVersion "23.1.7779620"
-	compileSdkVersion 30
+	compileSdkVersion 31
```

Comment:
This is due to https://developer.android.com/jetpack/androidx/releases/work requiring version 31

Reply:
Every November, Google increments the required SDK version to be allowed to upload to the Play Store.
- as of November 2021 only compileSdkVersion 29 and later can be uploaded
- as of November 2022 only compileSdkVersion 30 and later can be uploaded
- we would have increased compileSdkVersion to 31 by November anyway, it is fine to do so here.